collections/_article/progress-report-web-export-in-4-3.md

```diff
@@ -0,0 +1,281 @@
+| Catburglar | [Play](https://adamscott.github.io/catburglar-main-thread-samples/){:target="_blank"} | [Link](https://johngabrieluk.itch.io/catburglar){:target="_blank"} |
+
+_Please note: we already know that there's some issues running Platformer 2D on Apple devices, but these issues seem related to tilemap rendering issues with ANGLE over Metal as OpenGL replacement on macOS._
+
+Many thanks to the awesome [Catburglar](https://johngabrieluk.itch.io/catburglar) team (John Gabriel, Jerico Landry,
+Kyveri, Sacha Feldman, and Carrie Drovdlic). Without this [open-source project](https://bitbucket.org/JohnGabrielUK/catburglar/),
+made for the [63rd](https://itch.io/jam/godot-wild-jam-63) [Godot Wild Jam](https://godotwildjam.com/), I wouldn't have
+solved as many bugs as I did. Please try it out! The audio work, in particular, sounds incredible.
+
+### Here be dragons
+Even if the odds are that sample playback will be a part of Godot 4.3 release, it was developed very quickly,
+and even if it works pretty well, if you ask me, it still is pretty much a prototype. But we can't just release Godot 4.3
+without proper audio support for single-threaded releases.
+
+So it has been decided that it should be treated as _experimental_. More work will be done before the release of Godot 4.4
+in order to stabilize the API and how it works under the hood.
+
+### Call to action
+But even before thinking about Godot 4.4, I must call for your help! We need people to test out [the sample playback <abbr title="Pull Request">PR</abbr>](https://github.com/godotengine/godot/pull/85939)
+until the release.
+
+If you encounter any issue while testing my PR or while testing out the single-threaded samples builds, please report
+them [here](https://github.com/godotengine/godot/pull/85939).
+
+Finally, if you still intend to continue using threads on the Web platform, you can! And the next feature should aid you
+setup your game online.
+
+## Easy <abbr title="Cross-Origin-Opener-Policy">COOP</abbr>-<abbr title="Cross-Origin-Embedder-Policy">COEP</abbr> <abbr title="Progressive Web App">PWA</abbr> for threaded builds
+With this new feature, if you still need to export your game with thread support, you'll are able to activate the necessary
+<abbr title="Cross-Origin-Opener-Policy">COOP</abbr>/<abbr title="Cross-Origin-Embedder-Policy">COEP</abbr> headers
+without having to modifiy the server's header responses. You'll need to export your game as a Progressive Web App though.
+
+![Ensure Cross-Origin Isolation Headers export option](/storage/blog/progress-report-web-export-in-4-3/ensure-cross-origin-isolation-headers.webp)
+
+This is because this export option setup the Service Worker, included with the <abbr title="Progressive Web App">PWA</abbr>
+build, to intercept the requests from the server and inject those missing
+<abbr title="Cross-Origin-Opener-Policy">COOP</abbr>-<abbr title="Cross-Origin-Embedder-Policy">COEP</abbr> headers.
```

Comment:
```suggestion
This is because this export option sets up the Service Worker, included with the <abbr title="Progressive Web App">PWA</abbr>
build, to intercept the requests from the server and inject those missing
<abbr title="Cross-Origin-Opener-Policy">COOP</abbr>-<abbr title="Cross-Origin-Embedder-Policy">COEP</abbr> headers.
```